src/devices/Tca9548a/samples/Program.cs

```diff
@@ -0,0 +1,111 @@
+                    {
+                        I2cDevice bnoI2cDevice = bus.CreateDevice(device);
+                        Bno055Sensor bno055Sensor = new Bno055Sensor(bnoI2cDevice);
+                        Console.WriteLine($"Id: {bno055Sensor.Info.ChipId}, AccId: {bno055Sensor.Info.AcceleratorId}, GyroId: {bno055Sensor.Info.GyroscopeId}, MagId: {bno055Sensor.Info.MagnetometerId}");
+                        Console.WriteLine($"Firmware version: {bno055Sensor.Info.FirmwareVersion}, Bootloader: {bno055Sensor.Info.BootloaderVersion}");
+                        Console.WriteLine($"Temperature source: {bno055Sensor.TemperatureSource}, Operation mode: {bno055Sensor.OperationMode}, Units: {bno055Sensor.Units}");
+                        Console.WriteLine($"Powermode: {bno055Sensor.PowerMode}");
+                        var calibrationStatus = bno055Sensor.GetCalibrationStatus();
+                        Console.WriteLine($"Calibration Status : {calibrationStatus}");
+                        var magneto = bno055Sensor.Magnetometer;
+                        Console.WriteLine($"Magnetometer X: {magneto.X} Y: {magneto.Y} Z: {magneto.Z}");
+                        var gyro = bno055Sensor.Gyroscope;
+                        Console.WriteLine($"Gyroscope X: {gyro.X} Y: {gyro.Y} Z: {gyro.Z}");
+                        var accele = bno055Sensor.Accelerometer;
+                        Console.WriteLine($"Acceleration X: {accele.X} Y: {accele.Y} Z: {accele.Z}");
+                        var orien = bno055Sensor.Orientation;
+                        Console.WriteLine($"Orientation Heading: {orien.X} Roll: {orien.Y} Pitch: {orien.Z}");
+                        var line = bno055Sensor.LinearAcceleration;
+                        Console.WriteLine($"Linear acceleration X: {line.X} Y: {line.Y} Z: {line.Z}");
+                        var gravity = bno055Sensor.Gravity;
+                        Console.WriteLine($"Gravity X: {gravity.X} Y: {gravity.Y} Z: {gravity.Z}");
+                        var qua = bno055Sensor.Quaternion;
+                        Console.WriteLine($"Quaternion X: {qua.X} Y: {qua.Y} Z: {qua.Z} W: {qua.W}");
+                        var temp = bno055Sensor.Temperature.DegreesCelsius;
+                        Console.WriteLine($"Temperature: {temp} °C");
+                    }
+                    else if (device == Bmp180.DefaultI2cAddress)
+                    {
+                        using I2cDevice bmpDevice = bus.CreateDevice(Bmp180.DefaultI2cAddress);
+                        using Bmp180 i2cBmp280 = new(bmpDevice);
+                        // set samplings
+                        i2cBmp280.SetSampling(Sampling.Standard);
+                        // read values
+                        Temperature tempValue = i2cBmp280.ReadTemperature();
+                        Console.WriteLine($"Temperature: {tempValue.DegreesCelsius:0.#}\u00B0C");
```

Comment:
Consider instead to create I2cDevice implementation which works more or less as following:

```csharp
Tca9548A i2cMux = new Tca9548A(i2cBus);
I2cBus channel1Bus = i2cMux.GetBus(1);
using Bmp280 bmp280 = new (channel1Bus.CreateDevice(Bmp180.DefaultI2cAddress));

// when bmp280.Whatever() is called then virtual I2cDevice switches to the right channel automatically and Reads/Writes to that i2cDevice
```